src/index.js

```diff
@@ -6,6 +6,9 @@
   For simple types, like number, boolean, string and etc.: use lower-case,
   For complex types, like Object, Array and etc.: use first letter in upper-case
 **/
+
+import logger from './logger'
```

Comment:
Is this necessary have `logger` import on the top? Because for consistency first must be imported `React`.

Reply:
This is the app entry point file, so if we want to be sure to capture ALL potential console logging, the logger needs to initialize (starting the capture of `console.log` etc) before all other imports.

However, the risk of moving that import below the React imports is quite minimal.  A few `console` logs may get to the browser console (from the modules imported before the logger) before **logger.js** can setup the capture and be configured by **readConfiguration**.

I'll move it down to maintain "import React first" consistency.